ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -369,6 +375,56 @@ public sealed class ArchiWebHandler : IDisposable {
 			return result;
 		}
 
+		[PublicAPI]
+		public async Task<int?> GetPointsBalance() {
+			(bool success, string? accessToken) = await CachedAccessToken.GetValue().ConfigureAwait(false);
+
+			if (!success || string.IsNullOrEmpty(accessToken)) {
+				return null;
+			}
+
+			KeyValue? response = null;
+
+			for (byte i = 0; (i < WebBrowser.MaxTries) && (response == null); i++) {
+				using WebAPI.AsyncInterface loyaltyRewardsService = Bot.SteamConfiguration.GetAsyncWebAPIInterface(LoyaltyRewardsService);
+
+				loyaltyRewardsService.Timeout = WebBrowser.Timeout;
+
+				try {
+					response = await WebLimitRequest(
+						WebAPI.DefaultBaseAddress,
+
+						// ReSharper disable once AccessToDisposedClosure
+						async () => await loyaltyRewardsService.CallAsync(
+							HttpMethod.Get, "GetSummary", args: new Dictionary<string, object>(2, StringComparer.Ordinal) {
+								{ "access_token", accessToken! },
+								{ "steamid", Bot.SteamID.ToString(CultureInfo.InvariantCulture) }
+							}
+						).ConfigureAwait(false)
+					).ConfigureAwait(false);
+				} catch (TaskCanceledException e) {
+					Bot.ArchiLogger.LogGenericDebuggingException(e);
+				} catch (Exception e) {
+					Bot.ArchiLogger.LogGenericWarningException(e);
+				}
+			}
+
+			if (response == null) {
+				Bot.ArchiLogger.LogGenericWarning(string.Format(CultureInfo.CurrentCulture, Strings.ErrorRequestFailedTooManyTimes, WebBrowser.MaxTries));
+				return null;
+			}
+
+			KeyValue pointsInfo = response["summary"]["points"];
+
+			if (pointsInfo == KeyValue.Invalid) {
```

Comment:
Is this check needed because of `AsInteger` call below and defaulting to `int.MinValue`?

Reply:
The original purpose was to validate both of those and log if they're invalid in the response. I was also about to add it myself but yeah could be added in advance.